test/integration/cart_integration_test.rb

```diff
@@ -174,18 +175,31 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
   end
 
   test 'a user can checkout once logged in' do
-    skip
-    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    item = Item.create(
+      title: 'coffee',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.